### PR TITLE
Updated Map/map.pyde file with correct Indentation

### PR DESCRIPTION
**Pull request by Prabhnith**

### CODE REVIEW

File: mode/examples/Basics/Math/Map/Map.pyde [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'''
+ Map.
+ 
+ Use the map() function to take any number and scale it to a new number 
+ that is more useful for the project that you are working on. For example, use the
+ numbers from the mouse position to control the size or color of a shape. 
+ In this example, the mouse’s x-coordinate (numbers between 0 and 360) are scaled to
```
Comment (jdf) [inline mode/examples/Basics/Math/Map/Map.pyde]: In this example, the mouse’s x-coordinate (a number between 0 and 360) is scaled to

File: mode/examples/Basics/Math/Map/Map.pyde [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ new numbers to define the color and size of a circle.
```
Comment (jdf) [inline mode/examples/Basics/Math/Map/Map.pyde]: a new number to

File: mode/examples/Basics/Math/Map/Map.pyde [hunk 3]
```diff
@@ -0,0 +1,27 @@
+'''
+ Map.
+ 
+ Use the map() function to take any number and scale it to a new number 
+ that is more useful for the project that you are working on. For example, use the
+ numbers from the mouse position to control the size or color of a shape. 
+ In this example, the mouse’s x-coordinate (a number between 0 and 360) is scaled to a
+ new numbers to define the color and size of a circle.
```
Comment (jdf) [inline mode/examples/Basics/Math/Map/Map.pyde]: s/numbers/number

**Comment by Prabhnith** [comment]
any modification required??

File: mode/examples/Basics/Math/Map/Map.pyde [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ In this example, the mouse’s x-coordinate (a number between 0 and 360) is scaled to a
+ new number to define the color and size of a circle.
+ 
+ Example ReWritten in Python By: Prabhjot Singh (NITH)
+ example original in Java mode examples: Math->Maps
+'''
+def setup():
+    size(640, 360)
+    noStroke()
+
```
Comment (GoToLoop) [inline mode/examples/Basics/Math/Map/Map.pyde]: Just 1 extra space between the end of `def setup(): ` and the start of `def draw():`.

**Pull request merged**